### PR TITLE
MREs

### PR DESCRIPTION
**Pull request by 0xRainy**
## About The Pull Request
Adds MREs to the game!

The entree and snack have similar effects to heated donk pockets (omnizine) and the rest is just normal items/food.

Icons from TGMC.

Added them to some loot tables. (Maintenance/Lavaland)
Also adds Coffee Instant to the game.

## Why It's Good For The Game
MREs, nice.

## Changelog
:cl:
**add:** MRE storage item and accompanying food items
**add:** Coffee Instant
**add:** MREs to maintenance shaft and lavaland loot tables
**imageadd:** added some icons and images for the MRE, food, and coffee instant
/:cl:

### CODE REVIEW

**Comment by Kexerval** [comment]
Nice hiss

**Comment by 0xRainy** [comment]
Finished this up, added MREs to mining redemption machine, and cargo orders.

It now has poison depending on the age of the MRE or omnizine for fresh ones.

Feel free to add more quotes and MRE types!

File: code/modules/food_and_drinks/food/snacks.dm [hunk 1]
```diff
@@ -92,6 +93,11 @@ All foods are distributed among various categories. Use common sense.
 		to_chat(user, "<span class='warning'>None of [src] left, oh no!</span>")
 		qdel(src)
 		return FALSE
+
+	if(package)
+		to_chat(M, "<span class='warning'>How do you expect to eat this with the package still on?</span>")
```
Comment (prodirus) [inline code/modules/food_and_drinks/food/snacks.dm]:
```suggestion
		to_chat(M, "<span class='warning'>How do you expect to eat this while it is still in the package?</span>")
```
Maybe this sounds better?

File: tgstation.dme [hunk 2]
```diff
@@ -594,6 +594,7 @@
 #include "code\FulpstationCode\wintercoats.dm"
 #include "code\FulpstationCode\halloween_costumes\joseph_joestar.dm"
 #include "code\FulpstationCode\halloween_costumes\outfits.dm"
+#include "code\FulpstationCode\halloween_costumes\phantom_maid.dm"
```
Comment (prodirus) [inline tgstation.dme]: Where is this from? Is this relevant to this change?

File: code/modules/food_and_drinks/food/snacks.dm (hunk 1 above)
Comment (0xRainy) [reply code/modules/food_and_drinks/food/snacks.dm]: Sure, making change.

File: tgstation.dme (hunk 2 above)
Comment (0xRainy) [reply tgstation.dme]: It's not. Will attempt to fix.
